### PR TITLE
Norway: New Icons

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: app/src/main/res/drawable/ic_hedvig_h.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="64.31dp"
+    android:height="64.31dp"
+    android:viewportWidth="64.31"
+    android:viewportHeight="64.31">
+  <path
+      android:pathData="M32.16,0A32.16,32.16 0,1 0,64.31 32.16,32.24 32.24,0 0,0 32.16,0ZM32.16,59.83a27.8,27.8 0,1 1,0 -55.59A28,28 0,0 1,60.08 32.16,27.67 27.67,0 0,1 32.16,59.83Z"
+      android:fillColor="#fafafa"/>
+  <path
```
Comment (MathiasOhrstrom) [inline app/src/main/res/drawable/ic_hedvig_h.xml:9]: Have you run avocado on this? 🥑

File: app/src/main/res/drawable/ic_launcher_foreground.xml [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
-    <path android:pathData="M868.4 760.3h-65c0-53.2-43.3-96.4-96.4-96.4s-96.4 43.3-96.4 96.4h-65c0-89 72.4-161.5 161.5-161.5 88.8 0 161.3 72.4 161.3 161.5z" android:fillColor="#651FFF"/>
-    <path android:pathData="M545.4 760.3h65v155.9h-65z" android:fillColor="#FF8A80"/>
-    <path android:pathData="M545.4 470.1h65v290.1h-65z" android:fillColor="#0F007A"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="108dp"
+    android:height="108dp"
+    android:viewportWidth="512"
+    android:viewportHeight="512">
+  <path
+      android:pathData="M256,384C185.249,384 128,326.751 128,256C128,185.249 185.249,128 256,128C326.751,128 384,185.249 384,256C384,326.751 326.751,384 256,384ZM256,145.283C194.97,145.283 145.283,194.971 145.283,256C145.283,317.03 194.97,366.717 256,366.717C317.029,366.717 366.717,317.03 366.717,256C366.717,194.971 317.029,145.283 256,145.283Z"
```
Comment (MathiasOhrstrom) [inline app/src/main/res/drawable/ic_launcher_foreground.xml:7]: 🥑

File: app/src/main/res/drawable/ic_hedvig_h.xml (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/res/drawable/ic_hedvig_h.xml:9]: 🥑

File: app/src/main/res/drawable/ic_launcher_foreground.xml (hunk 2 above)
Comment (hedvigoscar) [reply app/src/main/res/drawable/ic_launcher_foreground.xml:7]: 🥑

**Pull request merged**